replication-api-impl/src/main/java/org/codice/ditto/replication/api/impl/ReplicatorImpl.java

```diff
@@ -136,15 +121,15 @@ public void init() {
                 executeSyncRequest(syncRequest);
               }
             } catch (InterruptedException e) {
-              LOGGER.trace("InterruptedException in executor. This is expected during shutdown.");
+              LOGGER.debug("InterruptedException in executor. This is expected during shutdown.");
               Thread.currentThread().interrupt();
               break;
             }
           }
         });
 
-    LOGGER.trace(
-        "Successfully configured the single-thread scheduler to execute sync requests from the queue");
+    LOGGER.info(
```

Comment:
❓ info? This seems like developer's info and not customers so debug would be more appropriate